addons/account/models/account_move.py

```diff
@@ -1412,6 +1412,7 @@ def _compute_payments_widget_reconciled_info(self):
                         'account_payment_id': counterpart_line.payment_id.id,
                         'payment_method_name': counterpart_line.payment_id.payment_method_line_id.name,
                         'move_id': counterpart_line.move_id.id,
+                        'is_payment_or_bank': bool(counterpart_line.payment_id or counterpart_line.move_id.statement_line_id or counterpart_line.move_id.move_type not in ['in_refund', 'out_refund']),
```

Comment:
I think we should remove the check on `move_type`.

(By the way, if you have the check on move_type, the two other conditions are redundant, since moves linked to payments and bank statement lines always have `move_type == 'entry'`.)

Reply:
okay, thanks for picking up on that! I just asked CHKL and he said indeed we should show `paid` in that case.
So the original version of your fix actually makes sense (where you just check that move_type not in ['in_refund', 'out_refund'])

Sorry for the back-and-forth, feel free to change it back to what it was before I reviewed and I'll r+ :smile: